wire-ios/Wire-iOS/Sources/Managers/DatadogWrapper.swift

```diff
@@ -189,7 +189,11 @@ public class DatadogWrapper {
         message: String,
         error: Error? = nil,
         attributes: [String: Encodable]? = nil
-    ) {}
+    ) {
+        #if DEBUG
+        print("[Datadog OFF] \(message)")
+        #endif
+    }
```

Comment:
Is this so we can see DD logs when debugging without modifying the build config?

Reply:
that was the idea, but didn't work, I'll remove from this.
And re-apply to develop, it misses a -DDEBUG in WireCommon